### DISCUSSION TITLE
Form Part missing in one of my tennants

### DISCUSSION THREAD

**Discussion by bashuss** [OrchardCMS/OrchardCore#11459] [Q&A]
I have a second tenant running on my orchard core, which is currently runny Version 1.1. 

This tenant had the Forms Module activated in features, but there was no Form content type defined as in my other tennant. Also when trying to create a Form content type from scratch inside the admin menu, I realized that there is not Form content part available on the tenant. So I cannot rebuild.

Is there a way to get that tenant updated? Obviously something is missing there. I checked features, but cannot find the problem.
It has been quite a while, since I last worked on that tenant and orchard was still in beta stage...

best regards, 
Bernd

**Comment by ns8482e** [comment]
You might have the `Forms` module not enabled

**Reply by bashuss** [reply]
No, Forms module is activated in all my tenants.

**Comment by bashuss** [comment]
The reason might be, that this tenant was originally not set to persisting content definitions in file system.
It seems like that some entries in ContentDefinition.json are missing compared to other tenants but only the entries persisted there are used. I have not found a way to generate the missing entries. Deactivating and Reactivating Forms module does not change anything.

**Reply by bashuss** [reply]
I solved this for now by manually copying the entries I needed from other tenants. But this only works as long as I do have a tenant which has the proper entries. I might create a fresh tenant to generate the entries, but this is a bit of an extra overkill to the overkill of fixing this manually. Should this be a bug report or still a discussion?
